mobify/nightwatch-commands
# Add linting

This PR adds jshint and jscs to nightwatch-commands, ensuring that the code adheres to our standard.

Status: **Ready for merge**

Reviewers: @kbingman 
## Changes
- added Gruntfile
- added jshint/jscs and config files
- cleaned up all errors 
## How to test-drive this PR
- checkout branch
- run `grunt`
- verify there are no errors
- run `npm test`
- ensure the tests run
